src/__tests__/resolve-info.test.ts

```diff
@@ -1234,6 +1234,37 @@ describe("resolver info", () => {
                   }
               `);
       });
+
+      test("shouldInclude function is there and it works", async () => {
+        const doc = parse(`
+          query ($var: Boolean!) {
+            node(id: "tag:1") {
+                ... on Tag @skip(if: $var) {
+                  id
+                }
+                ... on Image {
+                  width
+                }
+              }
+            }
+        `);
+        const rootValue = { root: "val" };
+
+        await executeQuery(schema, doc, rootValue, null, {
+          var: true
+        });
+        const validationErrors = validate(schema, doc);
+        if (validationErrors.length > 0) {
+          console.error(validationErrors);
```

Comment:
instead of logging, add an expect statement to have validation errors are empty.